### PR TITLE
Test caseapi lookup for contact centre querying

### PR DESCRIPTION
**Pull request by hbrace**
# Motivation and Context

These tests are to ensure the caseapi endpoints supply the correct response when queried.
# What has changed

Case lookup feature has been added, with a number of scenarios. These tests are basic just to test functionality of the endpoints.
Most of the tests that query data use existing case steps to load the sample and get the details from rabbit.
The England sample file has had some UPRNs added to ensure all endpoints are covered.
# How to test?

mvn clean install off this branch https://github.com/ONSdigital/census-rm-case-api/tree/contact-centre-case-lookup and do a make up in docker.
Run the acceptance tests.
# Links

https://trello.com/c/Tg697scy/633-rmc-139-provide-case-look-up-for-cc-13
# Screenshots (if appropriate):

### CODE REVIEW

**Comment by hbrace** [comment]
[RMC-139 Provide case look up for CC (13)](https://trello.com/c/Tg697scy/633-rmc-139-provide-case-look-up-for-cc-13)

**Review by liamtoozer — CHANGES_REQUESTED** [review]
Think the `kubernetes.env` file will also updating with case-api env variables to correctly run in k8s.

File: acceptance_tests/features/case_look_up.feature [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Feature: Case look up for the contact centre
+
+  Scenario: Find case by caseId
+    Given sample file "sample_input_england_census_spec.csv" is loaded
+    Then the new cases are emitted to Respondent Home
+    And a case can be retrieved from the caseapi service
+
+  Scenario: Check non-existent caseId returns an empty list
```
Comment (AdeHarrison) [inline acceptance_tests/features/case_look_up.feature]: change "returns an empty list" and ready to approve

**Review by AdeHarrison — APPROVED** [review]
LGTM

**Review by NickGrantX — APPROVED** [review]
LGTM 👍 

Acceptance tests passing

**Review by liamtoozer — APPROVED** [review]

**Pull request merged**